layouts/shortcodes/observability_pipelines/configure_existing_pipelines/source_env_vars/amazon_data_firehose.md

```diff
@@ -0,0 +1,3 @@
+- Amazon Data Firehose address
+    -  The Observability Pipelines Worker listens to this socket address to receive logs from Amazon Data Firehose.
```

Comment:
```suggestion
    - The Observability Pipelines Worker listens to this socket address to receive logs from Amazon Data Firehose.
```
Not sure this would show up in the HTML anyway, but just in case